### ISSUE TITLE
Create / Modify Timeline UI

### ISSUE THREAD

**Issue by hanej** [google/timesketch#1779]
**Is your feature request related to a problem? Please describe.**
When we're in the middle of an incident, it would be much easier to create new timelines or add new data to existing timelines within the UI.  Currently we have to keep all of the data stored in CSV files locally, modify them, delete the timeline from the UI, re-upload the new timeline.  This causes problems with saved searches too as they are removed from saved searches and you have to go back and fix them.

**Describe the solution you'd like**
It would be nice to open a timeline in a basic Google Sheets type UI to create new timelines or edit existing ones.  End users could also create blank templates of common data that they are importing to keep the type of data consistent across investigations.

**Comment by jaegeral** [comment]
Two things on that: editing already existing events could be very controversial as in it was never a key concept of TImesketch to do that afaik.

For the second thing to add events as response progresses, there is an API to that and we even have a notebook to add a single Event to the sketch specific timeline: https://github.com/google/timesketch/blob/master/notebooks/add_a_single_event_to_a_sketch.ipynb. This has not been added to the UI but is in the API: https://github.com/google/timesketch/pull/692

So basically the idea is that you do not have to keep your data in CSVs, but work in Timesketch.

As a replacement to delete events, you can always hide them from the UI or apply a label like "false_alarm" and create your saved searches to ignore those events.

There is also some related work: https://github.com/google/timesketch/issues/1586

https://github.com/google/timesketch/issues/1292
https://github.com/google/timesketch/issues/819

**Comment by berggren** [comment]
Another thought here - we already support appending data to exisiting timelines so this might cover usecase 1. We don't support de-dup because this is too expensive in the elasticsearch side.

Re adding events manually - this is indeed supported already, but haven't made it to the UI yet. Tracking in #1940

**Comment by hanej** [comment]
Just came across this today.  This is really all you need for a UI.

https://docs.trymito.io/

**Comment by joachimmetz** [comment]
@hanej have you read https://docs.trymito.io/misc./terms-of-service ?

**Comment by jaegeral** [comment]
There was little progress on this issue, so I am going top close it out, @hanej let me know if you have further thoughts or if we can help with specific workflows on how to make them better. The general TLDR of the issue sounds like a different tool is needed vs improve some things in TImesketch
